data_steward/tools/run_achilles_and_export.py

```diff
@@ -32,15 +41,23 @@ def main(args):
         formatter_class=argparse.RawDescriptionHelpFormatter)
     parser.add_argument(
         '--bucket',
-        default=get_hpo_bucket('nyc'),
-        help=
-        'Identifier for the bucket. Output tables will be prepended with {hpo_id}_.'
+        default='',
+        help='Identifier for the bucket where, achilles files will be copied to'
     )
     parser.add_argument(
         '--folder',
         default='',
-        help=
-        'Identifier for the folder in which achilles results sit. Output tables will be '
-        'prepended with {hpo_id}_.')
+        help='Identifier for the folder in which achilles results sit.',
+        required=True)
+    parser.add_argument(
+        '--transfer_bucket',
+        default='',
+        help='Achilles Transfer bucket name',
+    )
+    parser.add_argument(
+        '--data_stage',
```

Comment:
I made it optional because we only needed combined datasets reports to be copied to transfer bucket. I am making a change which makes it a required but not using the options.